tests/ui/test_pvc_deployment_ui.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
```

Comment:
name of this file can be just test_deployment_ui as there is nothing done specific to pvc here.